### DISCUSSION TITLE
[FORD] 注釈付加作業協力求ム

### DISCUSSION THREAD

**Discussion by Enehentaro** [Enehentaro/droplets_simulation#124] [Announcements]
現在、ひそかにドキュメント生成作業を行っています。

現状こんな感じ：
https://enehentaro.github.io/droplets_simulation/

生成自体はできるようになったのですが、FORD用の注釈がほとんど記述できていないのでみすぼらしい感じになってます。
気が向いたらでいいので、注釈付加作業に協力してくれるとうれしいです。
注釈付加作業は、どのブランチでやっても大丈夫です。あとで`FORD-docs`ブランチにマージすればいいので。

注釈の書き方は[こちら](https://qiita.com/implicit_none/items/8fc065df6a295183b141#%E6%B3%A8%E9%87%88%E3%81%AB%E3%82%88%E3%82%8B%E6%83%85%E5%A0%B1%E3%81%AE%E8%BF%BD%E5%8A%A0)

**Comment by hirotori** [comment]
SCTfile_reader.f90の注釈を直しました.

> 注釈の書き方は[こちら](https://qiita.com/implicit_none/items/8fc065df6a295183b141#%E6%B3%A8%E9%87%88%E3%81%AB%E3%82%88%E3%82%8B%E6%83%85%E5%A0%B1%E3%81%AE%E8%BF%BD%E5%8A%A0)

リンク先の注釈のうち, [FORDで利用出来る記法](https://qiita.com/implicit_none/items/8fc065df6a295183b141#ford%E3%81%A7%E5%88%A9%E7%94%A8%E3%81%A7%E3%81%8D%E3%82%8B%E8%A8%98%E6%B3%95) を参考に書き直しました. `@note`~`@endnote` は記事の通り, 他の注釈と1行開けないとうまく認識しないようですのでご注意を.
```
subroutine hoge(huga)
    !! すごいサブルーチン
    !! version: 1.1.0
    !!                                     <-----ここは1行開ける
    !!@note
    !! とてもすごい.
    !!@endnote
    以下つづく...
```
